docker/ci/Dockerfile

```diff
@@ -23,7 +23,19 @@ RUN set -x \
 ENV BISON_VERSION 3.0.5
 RUN curl -L https://ftp.gnu.org/gnu/bison/bison-$BISON_VERSION.tar.gz | tar xzvf - \
     && cd bison-$BISON_VERSION && ./configure --prefix=/usr && make all install \
-    && rm -rf bison-$BISON_VERSION
+    && rm -rf bison-$BISON_VERSION && cd ..
+
+# install cmake 3.26
+ENV CMAKE_VERSION 3.26.3
+RUN curl -L https://github.com/Kitware/CMake/releases/download/v$CMAKE_VERSION/cmake-$CMAKE_VERSION-linux-x86_64.tar.gz | tar xzvf - \
+ && yes | cp -fR cmake-$CMAKE_VERSION-linux-x86_64/* /usr
```

Comment:
If you don't need it, could you remove unnecessary files to reduce a image size?